helper/fields/data.go

```diff
@@ -18,7 +18,16 @@ type FieldData struct {
 func (d *FieldData) Validate() error {
 	var result *multierror.Error
 
-	// Scan for missing required fields
+	// Check that either image or pod_manifest was passed in but not both or not at all
+	_, podManifestExists := d.Raw["pod_manifest"]
+	_, imageExists := d.Raw["image"]
+
+	if podManifestExists == imageExists {
+		result = multierror.Append(result, fmt.Errorf(
+			"Either field image or field pod_manifest is required but not both"))
```

Comment:
I am a bit confused by this? Not sure this works, as if they are both false this if statment would pass as well?

Maybe `if podManifestExists  && imageExists  {...}` would be a bit more clear.

Also I would rephrase the error msg to something like:
> Both field image and pod manifest were passed. Only one is required.
And also print out the values of both vars.

Reply:
You need to pass either an image or a pod manifest. Passing nothing or passing both is an error.

Reply:
Then just the error msg should be reworked.

Reply:
@LiliC Fixed the error message as suggested but not printing out the variables. It would be a lot of noise with the `pod_manifest` being potentially huge, don't you think?